### ISSUE TITLE
关于指标结果的一些问题

### ISSUE THREAD

**Issue by 123dddwn** [csslc/PiSA-SR#42]
您好，非常精彩的工作，但是我有一些疑问关于训练时候的参数设置，以下是我训练时候的参数设置，不知是否有不合适的地方，因为我复现出来的指标结果与论文中有一定的差距，而且在训练过程中的验证结果也显示出有很多的噪声和伪影。
期待您的回复！
CUDA_VISIBLE_DEVICES="0" accelerate launch train_pisasr.py \
    --pretrained_model_path="src/models/stable-diffusion-2-1-base" \
    --pretrained_model_path_csd="src/models/stable-diffusion-2-1-base" \
    --dataset_txt_paths="preset/training_datasets/gt_path.txt" \
    --highquality_dataset_txt_paths="preset/training_datasets/gt_selected_path.txt" \
    --dataset_test_folder="preset/test_datasets/RealSR_CenterCrop" \
    --learning_rate=5e-5 \
    --train_batch_size=4 \
    --prob=0.1 \
    --gradient_accumulation_steps=1 \
    --enable_xformers_memory_efficient_attention --checkpointing_steps 2000 \
    --seed 123 \
    --output_dir="experiments/train-pisasr" \
    --cfg_csd 7.5 \
    --timesteps1 1 \
    --lambda_lpips=2.0 \
    --lambda_l2=1.0 \
    --lambda_csd=1.0 \
    --pix_steps=4000 \
    --lora_rank_unet_pix=4 \
    --lora_rank_unet_sem=4 \
    --min_dm_step_ratio=0.02 \
    --max_dm_step_ratio=0.5 \
    --null_text_ratio=0.5 \
    --align_method="adain" \
    --deg_file_path="params.yml" \
    --tracker_project_name "PiSASR" \
    --is_module True \
